tfx/components/example_validator/component.py

```diff
@@ -68,6 +68,8 @@ class ExampleValidator(base_component.BaseComponent):
   def __init__(self,
                statistics: types.Channel = None,
                schema: types.Channel = None,
+               exclude_splits: Optional[List[Union[
+                   Text, data_types.RuntimeParameter]]] = None,
```

Comment:
could you double check if this data_types.RuntimeParameter works, can ExampleValidatorSpec parse it automatically?

Reply:
i removed RuntimeParameter for now.